src/main/kotlin/com/tealcube/minecraft/bukkit/mythicdrops/inventories/AnvilListener.kt

```diff
@@ -52,8 +53,7 @@ internal class AnvilListener(
     }
 
     private fun handleEarlySocketExtenderCheck(event: PrepareAnvilEvent) {
-        val anyAreSocketExtenders = event.inventory.contents.filterNotNull()
-            .any { it.displayName == settingsManager.socketingSettings.items.socketExtender.name.chatColorize() }
+        val anyAreSocketExtenders = event.inventory.anyDisplayName(settingsManager.socketingSettings.items.socketExtender.name.chatColorize())
```

Comment:
There's a slightly different fix required for this, which is to use the `SocketExtenderTypeManager` and compare against all of the known `SocketExtenderType`s.

Reply:
That would ultimately end up in something like the below:
```kotlin
val anyAreSocketExtenders = socketExtenderTypeManager.get().any { socketExtenderType ->
  event.inventory.anyDisplayName(socketExtenderType.socketExtenderStyleChatColorized)
}
```